src/main/java/kostka/moviecatalog/controller/AdministrationController.java

```diff
@@ -0,0 +1,133 @@
+import kostka.moviecatalog.service.ExternalCommentService;
+import kostka.moviecatalog.service.ExternalRatingService;
+import kostka.moviecatalog.service.rabbitmq.RabbitMqSender;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping(value = "/admin")
+public class AdministrationController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AdministrationController.class);
+    public static final String ADMIN_VIEW = "admin";
+    public static final String STATUS = "status";
+    public static final String INVALID_DTO = "Required fields are empty.";
+    private DbMovieService dbMovieService;
+    private RabbitMqSender rabbitMqSender;
+    private ExternalCommentService externalCommentService;
+    private ExternalRatingService externalRatingService;
+
+    @Autowired
+    public AdministrationController(final DbMovieService dbMovieService,
+                                    final RabbitMqSender rabbitMqSender,
+                                    final ExternalCommentService externalCommentService,
+                                    final ExternalRatingService externalRatingService) {
+        this.dbMovieService = dbMovieService;
+        this.rabbitMqSender = rabbitMqSender;
+        this.externalCommentService = externalCommentService;
+        this.externalRatingService = externalRatingService;
+    }
+
+    @GetMapping()
+    public String viewAdmin(final Model model) {
+        addModelAttributes(model, "");
+        return ADMIN_VIEW;
+    }
+
+    @PostMapping("/movie/create")
+    public String createMovie(final @Valid @ModelAttribute MovieDto dto,
+                              final BindingResult bindingResult,
+                              final Model model) {
+        LOGGER.info("create movie request");
+
+        if (bindingResult.hasErrors()) {
+            addModelAttributes(model, INVALID_DTO);
+            return ADMIN_VIEW;
+        }
+
+        try {
+            Movie movie = dbMovieService.createMovie(dto);
+            rabbitMqSender.sendToCreateElasticQueue(movie.getId().toString());
+            rabbitMqSender.sendUpdateRequestToQueue();
+        } catch (InvalidDtoException e) {
+            LOGGER.error("Creation of movie failed", e);
+            addModelAttributes(model, "Movie cannot be created.");
+            return ADMIN_VIEW;
+        }
+
+        addModelAttributes(model, "Movie is successfully created.");
+        return ADMIN_VIEW;
```

Comment:
After successful POST request you should do HTTP REDIRECT.
Otherwise there are issues with F5 refresh in browser.

Tip: Success message can then be handled via flash messages.